js/Store.js

```diff
@@ -45,7 +61,19 @@ function beginTurn(state, player) {
   };
 }
 
-function damage(state, cardId, amount) {
+function damagePlayer(state, player, amount) {
+  state = {
+    ...state,
+    life: state.life.update(player, m => m - amount),
+  };
+  return {
```

Comment:
Why does beginTurn do it that way? It seemed like when I changed beginTurn to just this, it didn't work, and I wondered why:

    function beginTurn(state, player) {
      return {
        ...state,
        turn: player,
        maxMana: state.maxMana.update(player, m => m + 1),
     };
}